staging/src/k8s.io/code-generator/cmd/conversion-gen/main.go

```diff
@@ -100,6 +100,11 @@ func main() {
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 	pflag.Parse()
 
+	// k8s.io/apimachinery/pkg/runtime contains a number of manual conversions,
+	// that we need to generate conversions.
+	// Adding this to InputDirs to ensure this package will be fully parsed.
+	genericArgs.InputDirs = append(genericArgs.InputDirs, "k8s.io/apimachinery/pkg/runtime")
```

Comment:
This seems.... hackish, even though this is the highest level we can call it.  I kind of expected this input dir to be automatic anyway because of dependencies - is that not the case (do we not scan import packages for conversions anymore?)

Reply:
Yes and no - we scan for dependencies, but not the whole dependency packages, but only the types that are referenced from explicitly requested packages. So we don't scan for functions or not-used types not used in explicitly requested packages.
https://github.com/kubernetes/gengo/blob/7fa3014cb28f99ce2d9e803bcd3efa7edd4effa9/parser/parse.go#L453

I extended the comment to make it explicitly stated.

Reply:
thanks